python/tidy/servo_tidy/tidy.py

```diff
@@ -483,6 +483,7 @@ def check_rust(file_name, lines):
             (r"\{[A-Za-z0-9_]+\};", "use statement contains braces for single import",
                 lambda match, line: line.startswith('use ')),
             (r"^\s*else {", "else braces should be on the same line", no_filter),
+            (r"[^\(][^$]\([ \t\r\v\f][^\(]", "extra space after (", no_filter),
```

Comment:
We don't have to explicitly make it ignore macros. This could be a bit more simpler. Since we're concentrating on function names, we can assume that there shouldn't be any space or `$` before the parentheses. Also, we don't want the vertical space, form feed and carriage return checks - just the space and tab should be fine :)